code/modules/reagents/reagent_containers/syringes.dm

```diff
@@ -158,6 +158,11 @@
 				if(!target.is_open_container() && !ismob(target) && !istype(target, /obj/item/weapon/reagent_containers/food) && !istype(target, /obj/item/slime_extract) && !istype(target, /obj/item/clothing/mask/cigarette) && !istype(target, /obj/item/weapon/storage/fancy/cigarettes))
 					user << "\red You cannot directly fill this object."
 					return
+				if(istype(target, /obj/item/clothing/mask/cigarette))

```

Comment:
Concur with Goof on this one. DIY Cigs that also cost like 2-3x would also be interesting, but that's also kinda outside of the scope of this PR =p